clang/lib/CodeGen/Targets/AArch64.cpp

```diff
@@ -399,7 +399,7 @@ AArch64ABIInfo::classifyArgumentType(QualType Ty, bool IsVariadic,
   return getNaturalAlignIndirect(Ty, /*ByVal=*/false);
 }
 
-ABIArgInfo AArch64ABIInfo::classifyReturnType(QualType RetTy,
+ABIArgInfo AArch64ABIInfo:: classifyReturnType(QualType RetTy,
```

Comment:
???

Reply:
:laughing: 

CI says it OK. Green checks all the way.